src/server.h

```diff
@@ -952,6 +952,7 @@ struct redisServer {
     long long stat_active_defrag_key_hits;  /* number of keys with moved allocations */
     long long stat_active_defrag_key_misses;/* number of keys scanned and not moved */
     size_t stat_peak_memory;        /* Max used memory record */
+    size_t stat_peak_memory_memkind;/* Max used memkind memory record */
```

Comment:
You don't want to initialize this value in initServer method ?

Reply:
That's good point. Fixed.